spec/views/events/participants.html.erb_spec.rb

```diff
@@ -48,5 +48,83 @@
+
+    render
+    expect(rendered).to have_css("img[src*='vegan']")
+    expect(rendered).to have_css("img[src*='vegeterian']")
+  end
+
+  it "should display eating habits column and print button when looging in as organizer" do
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    render
+    expect(rendered).to have_link(I18n.t('application_letters.print_accepted_with_eatings_habits'))
+    expect(rendered).to have_link(I18n.t('activerecord.methods.application_letter.eating_habits'))
+  end
+
+  it "should not display eating habits column and print button when looging in as pupil" do
+    sign_in(FactoryGirl.create(:user, role: :pupil))
+    render
+    expect(rendered).not_to have_link(I18n.t('application_letters.print_accepted_with_eatings_habits'))
+    expect(rendered).not_to have_link(I18n.t('activerecord.methods.application_letter.eating_habits'))
+  end
+
+
+  # it "sorts participants in the expected order by eating habits" do
+  #     @event = FactoryGirl.create(:event)
+  #     @user1 = FactoryGirl.create(:user, name: 'ghk')
+  #     @profile1 = FactoryGirl.create(:profile, user: @user1, birth_date: 15.years.ago)
+  #     @application1 = FactoryGirl.create(:application_letter_accepted, user: @user1, event: @event, )
+  #     @agreement1 = FactoryGirl.create(:agreement_letter, user: @user1, event: @event)
+
+  #     @user2 = FactoryGirl.create(:user, name: 'bba')
+  #     @profile2 = FactoryGirl.create(:profile, user: @user2, birth_date: 16.years.ago)
+  #     @application2 = FactoryGirl.create(:application_letter_accepted, user: @user2, event: @event, vegan: true, vegeterian: true, allergic: true)
+
+  #     @user3 = FactoryGirl.create(:user, name: 'eee')
+  #     @profile3 = FactoryGirl.create(:profile, user: @user3, birth_date: 19.years.ago)
+  #     @application3 = FactoryGirl.create(:application_letter_accepted, user: @user3, event: @event)
+  #     @agreement3 = FactoryGirl.create(:agreement_letter, user: @user3, event: @event)
+
+  #     @user4 = FactoryGirl.create(:user, name: 'ddd')
+  #     @profile4 = FactoryGirl.create(:profile, user: @user4, birth_date: 16.years.ago)
+  #     @application4 = FactoryGirl.create(:application_letter_accepted, user: @user4, event: @event)
+
+  #     @user5 = FactoryGirl.create(:user, name: 'bbb')
+  #     @profile5 = FactoryGirl.create(:profile, user: @user5, birth_date: 20.years.ago)
+  #     @application5 = FactoryGirl.create(:application_letter_accepted, user: @user5, event: @event)
+
+  #     @user6 = FactoryGirl.create(:user, name: 'abc')
+  #     @profile6 = FactoryGirl.create(:profile, user: @user6, birth_date: 16.years.ago)
+  #     @application6 = FactoryGirl.create(:application_letter_accepted, user: @user6, event: @event)
+  #     @agreement6 = FactoryGirl.create(:agreement_letter, user: @user6, event: @event)
+      
+
+  #     render.extra_params()
+  #     expect(rendered).to eq([@user2, @user4, @user6, @user1, @user5, @user3])
+  #   end
```

Comment:
What's the purpose of all this comment-code? Since we have git, I'd prefer to delete this for the moment if it is not a mistake to have it as a comment here.